### PR TITLE
Cannabis fakes

### PR DESCRIPTION
**Pull request by nurugger07**
This PR includes modules for cannabis related data. Around the world, this is a rapidly growing industry and it would be good to have these modules included in faker. 

I've added:

- [X] USAGE.md docs if applicable
- [X] CHANGELOG.md

### CODE REVIEW

**Comment by nurugger07** [comment]
@igas just wanted to follow up on this PR. Have you had a chance to look at it & discuss with the other maintainers? The ruby faker project has a similar implementation for cannabis.

https://github.com/stympy/faker/blob/master/lib/faker/default/cannabis.rb

File: lib/faker/cannabis.ex [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  Returns a Cannabis brand string
+
+  ## Examples
+
+      iex> Faker.Cannabis.En.brand()
+      "Evolab"
+      iex> Faker.Cannabis.En.brand()
+      "CI Wholesale"
+      iex> Faker.Cannabis.En.brand()
+      "Muy"
```
Comment (vbrazo) [inline lib/faker/cannabis.ex]: We work with 4 examples. Please double check all your doctests and add/remove when needed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 
 ### Added
 
+- Added: `Faker.Cannabis` [[@nurugger07][]]
```
Comment (vbrazo) [inline CHANGELOG.md]: Since you're first time contributor, you should include your name at the end of this file.

Comment (nurugger07) [reply CHANGELOG.md]: @vbrazo done :)

File: lib/faker/cannabis.ex (hunk 1 above)
Comment (nurugger07) [reply lib/faker/cannabis.ex]: @vbrazo all have 4 examples now

**Comment by nurugger07** [comment]
@vbrazo regarding the namespace comment: should other fakers like beer be under the `Faker.PopCulture` namespace?

**Comment by anthonator** [comment]
I don't know if I agree with adding a `PopCulture` namespace. Seems unnecessary to me.

**Comment by nurugger07** [comment]
> I don't know if I agree with adding a `PopCulture` namespace. Seems unnecessary to me.

I agree with @anthonator :)

**Comment by nurugger07** [comment]
Just checking on this

**Comment by igas** [comment]
Hi all,

this looks good to me. Namespace-wise we'll need one see #207 & #208. Having said that, coming with good categories will be hard. I think for now I would continue this path to not make it harder to contribute, and then bump to 1.0 with namespaces. Most likely with deprecations before that.

**Pull request merged**